src/Common/src/Microsoft/Win32/SafeHandles/SafeCloseIcmpHandle.cs

```diff
@@ -1,9 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.Win32.SafeHandles;
```

Comment:
Is this used by any other assemblies? If not, why not leave it in S.N.Utilities?